### ISSUE TITLE
Release Codex 0.1.8

### ISSUE THREAD

**Issue by veaceslavdoina** [codex-storage/nim-codex#983]
### Codex release - [Release v0.1.8](https://github.com/codex-storage/nim-codex/releases/tag/v0.1.8)

- [x] Deploy new smart contracts if required
- [x] Save contracts deployment artifacts
- [x] Update nim-codex to use new Marketplace contract
- [x] Prepare a release branch
- [x] Create a release
- [x] Post deployment
   - [x] Bootstrap nodes
   - [x] Storage nodes
   - [x] Discordbot
- [x] Update documentation
- [x] ~Announcement~

**Comment by veaceslavdoina** [comment]
New smart contracts were deployed
Tx: [`0x7952ab76c12b7e6e747f09fa9eb501117df9c963c9dcc93d092bcfade9dd0b3d`](https://explorer.testnet.codex.storage/tx/0x7952ab76c12b7e6e747f09fa9eb501117df9c963c9dcc93d092bcfade9dd0b3d)
Address: [`0x5Bd66fA15Eb0E546cd26808248867a572cFF5706`](https://explorer.testnet.codex.storage/address/0x5Bd66fA15Eb0E546cd26808248867a572cFF5706)

<details>
<summary>Pod logs</summary>

```shell
> deploy
> hardhat deploy

Downloading compiler 0.8.23
Downloading compiler 0.8.23
Warning: This declaration shadows an existing declaration.
  --> contracts/Marketplace.sol:65:5:
   |
65 |     MarketplaceConfig memory configuration,
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Note: The shadowed declaration is here:
  --> contracts/Marketplace.sol:91:3:
   |
91 |   function configuration() public view returns (MarketplaceConfig memory) {
   |   ^ (Relevant source part starts here and spans across multiple lines).


Compiled 23 Solidity files successfully
Deployed Marketplace with Groth16 Verifier at:
0x5Bd66fA15Eb0E546cd26808248867a572cFF5706

npm notice 
npm notice New major version of npm available! 9.6.7 -> 10.9.0
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.9.0>
npm notice Run `npm install -g npm@10.9.0` to update!
npm notice 
Done! Sleeping indefinitely...
```
</details>

**Comment by veaceslavdoina** [comment]
New Codex version was released and deployed on Testnet.
